pyqwikswitch/qwikswitch.py

```diff
@@ -231,9 +232,42 @@ def decode_imod(packet, channel=1):
             return None
     return None
 
+# byte 0:  0f = pir
+# byte 1:  firmware
+# byte 2 and 3:  number of seconds (in hex) that the PIR sends until a device should react.
 
+def decode_pir(packet):
```

Comment:
All `decode` methods should have the signature of `(packet, channel=1)` for it to work in Home Assistant [see here](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/sensor/qwikswitch.py#L48)